decidim-core/app/views/decidim/devise/shared/_login_boxes.html.erb

```diff
@@ -0,0 +1,28 @@
+<div class="login__box-container">
+  <%= link_to(decidim.new_user_session_path, class: "login__box") do %>
```

Comment:
Great that these are now clickable as a whole! So touch screens should now work great.

But there was a regression when this change was made, we lost the hover effect on the element that looks like a link:
![Login box hover](https://user-images.githubusercontent.com/864340/187230315-83538066-0516-46fa-9bce-b6be0479f832.png)

The screenshot has the hover state on the "Log in" link but there is no underline on the text.

Either of these solutions would be fine to fix this IMO:
1. Add the hover effect only to the blue "Log in" text element
2. Add the hover effect to the blue "Log in" text no matter which part of the link is hovered

Reply:
I've ended with an underlined blue text plus highlighting the full border. Is a similar way it's done elsewhere